### DISCUSSION TITLE
Dataset Access

### DISCUSSION THREAD

**Discussion by RobertPolicelli** [nyumets/nyumets.github.io#6] [General]
Hello, I was hoping to get access to the dataset. Any guidance on how to do so would be greatly appreciated!

**Comment by pooya-mohammadi** [comment]
I have the same problem, @RobertPolicelli do you have any updates?

**Comment by barhou29** [comment]
Hello NYUMets team! I’ve followed the instructions to access the NYUMets Brain Dataset on S3 using the provided access point ARN (arn:aws:s3:us-east-1:230249121367:accesspoint/external-users-read-only-ap) in us-east-1. Here’s what I’ve done: created an IAM user with AmazonS3ReadOnlyAccess, configured AWS CLI with the credentials, and attempted to list files using aws s3 ls on the access point. Despite these steps, I received an Access Denied error. I also tried a custom policy with explicit s3:ListBucket and s3:GetObject permissions on the access point, but the issue persisted. It seems there may be additional restrictions on the access point. Could you confirm if external users require further permissions or approval to access this dataset? Thanks in advance for any guidance!
